src/rules/extensions.js

```diff
@@ -147,24 +153,21 @@ module.exports = {
         const extensionForbidden = isUseOfExtensionForbidden(extension)
         if (extensionRequired && !extensionForbidden) {
           context.report({
-            node: source,
```

Comment:
Actually IIRC the highlighting might be better in editors/IDEs using the entire node instead of just the source, even though one could argue that the error is only in the `source` bit. is that why?

I'd like to merge this, just want to understand this change first.

Reply:
You're right, not sure why I wasn't using the `source` node still. I've added it back in and added back the line/column number references to the existing `import` and new `require` tests.